rest/data/project.go

```diff
@@ -312,6 +312,22 @@ func (pc *DBProjectConnector) GetProjectSettingsEvent(p *model.ProjectRef) (*mod
 	return &projectSettingsEvent, nil
 }
 
+func (pc *DBProjectConnector) TestProjectAlias(p *model.Project, alias string, includeDeps bool) ([]restModel.APIVariantTasks, error) {
```

Comment:
I'm sort of against calling this TestProjectAlias, because A) it automatically sounds like a test function, and B) it feels misleading because it isn't actually _testing_  anything for success or failure, it's just returning results to a user. Maybe GetProjectAliasResults? I realize there maybe isn't another good name so if you prefer this that's fine.

Reply:
renamed to GetProjectAliasResults